R/textstat_readability.R

```diff
@@ -76,6 +76,7 @@ textstat_readability.default <- function(x,
     stop(friendly_class_undefined_message(class(x), "textstat_readability"))
 }    
 
+#' @importFrom stringi stri_length
 #' @export
 textstat_readability.corpus <- function(x,
                                         measure = c("all", "ARI", "ARI.simple", "Bormuth", "Bormuth.GP",
```

Comment:
Are we going for the R's conventional naming like `Xy.z` for column and argument labels? I am OK with this, but need to how this naming convention applies to other functions.

Reply:
No we don't want to use that anywhere else, but I think I used the same names as from the **korpus** package. And didn't want to change them once they were used. We could, but I am thinking of separating the readability functions into their own package anyway, and could do it then.

This was not changed by the PR.

Reply:
OK. Separating complex analytic functions is right thing to do.